addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -366,10 +366,18 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
      */
     start: function () {
         const proms = [this._super(...arguments)];
+        const isColumn = el => el.tagName === "DIV" && el.parentElement.classList.contains("row");
+        const isEditableFormColumn = el => isColumn(el) && !el.closest(".s_website_form_field, .s_website_form_submit");
         // Disable text edition
         this.$target.attr('contentEditable', false);
-        // Make button, description, and recaptcha editable
-        this.$target.find('.s_website_form_send, .s_website_form_field_description, .s_website_form_recaptcha').attr('contentEditable', true);
+        // Identifiy editable elements of the form: buttons, description,
+        // recaptcha and columns which are not fields.
+        const formEditableSelector = ".s_website_form_send, .s_website_form_field_description, .s_website_form_recaptcha";
+        for (const formEditableEl of this.$target[0].querySelectorAll(`:scope ${formEditableSelector}, :scope .row > div`)) {
+            if (!isColumn(formEditableEl) || isEditableFormColumn(formEditableEl)) {
```

Comment:
Actually, why do we need functions for this? Feels strange to search for elements including columns then checking again if they are columns or not. Can there really be columns *inside* s_website_form_field? Are they not always columns themselves so you could search for `.row > div:not(field)` directly?

Reply:
As far as I know, the field can contain other columns (see: [form field default layout](https://github.com/odoo/odoo/blob/15.0/addons/website/static/src/xml/website_form_editor.xml#L39-L46)) So if I search for `.row > div:not(field)` only, they will become editable... I updated the code to remove the functions and moved the conditions to the selectors in a way to make them more readable :+1: